client/packages/invoices/src/InboundShipment/DetailView/ContentArea/columns.tsx

```diff
@@ -204,7 +205,7 @@ export const useInboundShipmentColumns = () => {
       [
         'numberOfPacks',
         {
-          Cell: PositiveNumberCell,
+          Cell: props => <NumberInputCell {...props} min={1} />,
```

Comment:
there are now 12 instances of `<NumberInputCell {...props} min={1} />` and only 14 usages of `NumberInputCell` in total. 

Why not just change the defaults of `NumberInputCell`? then you don't need to import React and create a new component here